### PR TITLE
python3 compatibility, better command line options

### PR DESCRIPTION
**Pull request by vhdirk**
This commit adds support for python3, and also makes the config file location configurable

### CODE REVIEW

**Comment by nickstenning** [comment]
Thanks for this -- it looks great. One question: is there a reason you added `imaprelay.py` here?

**Comment by vhdirk** [comment]
The only reason for adding that is so that you can run and test it from the project root without having to run `setup.py`. I tend to do that with all setuptools projects I work on...

**Comment by nickstenning** [comment]
Ok, I would tend to create a virtualenv for the project and `pip install -e .` within that virtualenv. It's not really needed. Would you mind rebasing it out?

**Comment by vhdirk** [comment]
Ok, fair enough

File: imaprelay/relay.py [hunk 1]
```diff
@@ -100,13 +100,13 @@ def loop(self, interval=30):
     def _open_connections(self):
         try:
             self.imap = make_imap_connection()
-        except (socket.error, imaplib.IMAP4.error):
+        except (socket.error, imaplib.IMAP4.error, OSError):
```
Comment (nickstenning) [inline imaprelay/relay.py]: What can potentially throw OSError here?

Comment (vhdirk) [inline imaprelay/relay.py]: I was too hasty: in python3, OSError is exactly the same as socket.error. This is redundant.

**Comment by nickstenning** [comment]
Awesome stuff. If we get rid of the redundant OSError I'll happily merge this and release a 0.0.6.

**Comment by vhdirk** [comment]
ok, sounds great.

**Comment by nickstenning** [comment]
I'm having a cleanup of open PRs on my repositories. I'd be very happy to look at this again if it were cleaned up (address review comments, remove WIP commits, etc.) but for now I'm going to close this. Feel free to reopen if you get a chance to update it!

**Pull request closed without merging**